modules/apps/commerce/commerce-product-asset-categories-web/src/main/java/com/liferay/commerce/product/asset/categories/web/internal/portlet/action/EditAssetCategoryCPAttachmentFileEntryMVCActionCommand.java

```diff
@@ -172,7 +172,8 @@ private void _updateCPAttachmentFileEntry(ActionRequest actionRequest)
 				displayDateDay, displayDateYear, displayDateHour,
 				displayDateMinute, expirationDateMonth, expirationDateDay,
 				expirationDateYear, expirationDateHour, expirationDateMinute,
-				neverExpire, titleMap, null, priority, type, serviceContext);
+				neverExpire, true, titleMap, null, priority, type,
```

Comment:
Why true here?

Reply:
This is the category images, doesn't really matter. I set it to true since it is the default value.